apps/backend/src/modules/auth/auth.service.ts

```diff
@@ -1,4 +1,9 @@
+import * as jose from "jose";
```

Comment:
You shouldn't import `jose` here. Check #40 for the implementation of token module